devel/cppcheck/Portfile

```diff
@@ -45,6 +46,9 @@ post-patch {
 
 compiler.cxx_standard       2011
 compiler.thread_local_storage yes
+# build/valueflow.cpp:10973:124: error: call
+# to deleted function 'toString' on macOS <10.12
+compiler.blacklist-append   {*gcc-[3-4].*} {clang < 900}
```

Comment:
Not that it hurts, but blacklisting pre-C++11 gccs is redundant. They are already excluded by two options specified above.